lib/ruby_smb/client/signing.rb

```diff
@@ -1,64 +0,0 @@
-module RubySMB
```

Comment:
Why did you pull the signing module out of the client?

Reply:
So it could be used by both the client and the server.